### PR TITLE
Fix: Add `aspect-ratio` fallback in Video widget for old browsers [ED-10100]

### LINKED ISSUE #21454: ✅ 🐞 Bug Report: Video widget appears half of the height when visited using Safari after 3.11.1 - [ED-10100]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Video widget heights are not shown correctly mainly in Safari, on some websites also in other browsers, after updating to 3.11.1. The new height is about 50% of initial height of embedded YouTube / Vimeo videos in some browsers, mainly Safari, compromising usability and design. On some websites, the same videos still show correctly in Chrome, Edge, Vivaldi, Opera at the same time. On other websites the same issue is also seen in Firefox and Chrome.

### Steps to reproduce

1. Open a page with Elementor version 3.11.1/3.11.2/3.11.3 containing an Elementor video widget (YouTube, Vimeo) in Safari.
2. Compare with views in other browsers (Chrome, Firefox, Vivaldi, ...)
3. Compare with Elementor version 3.10.2

### Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [ ] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

Server Environment |   |  
-- | -- | --
Operating System: | Linux |  
Software: | Apache/2.4.41 (Ubuntu) OpenSSL/1.1.1f mod_fcgid/2.3.9 |  
MySQL version: | (Ubuntu) v5.7.39 |  
PHP Version: | 8.0.28 |  
PHP Memory Limit: | 128M |  
PHP Max Input Vars: | 1000 |  
PHP Max Post Size: | 20M |  
GD Installed: | Yes |  
ZIP Installed: | Yes |  
Write Permissions: | All right |  
Elementor Library: | Connected

### PR DESCRIPTION
**Pull request by rami-elementor**
Fixes #21454

### CODE REVIEW

File: assets/dev/scss/frontend/widgets/video.scss [hunk 1]
```diff
@@ -18,6 +19,26 @@
...
+				height: 0;
+				padding-bottom: calc( 100% / var( --video-aspect-ratio ) );
+
+				iframe,
+				video {
+					position: absolute;
+					top: 0;
+					right: 0;
+					bottom: 0;
+					left: 0;
```
Comment (StyleShit) [inline assets/dev/scss/frontend/widgets/video.scss:38]: inset?

File: includes/widgets/video.php [hunk 2]
```diff
@@ -697,16 +697,16 @@ protected function register_controls() {
...
-					'43' => '4 / 3',
-					'32' => '3 / 2',
-					'11' => '1 / 1',
-					'916' => '9 / 16',
+					'169' => '1.77777',
+					'219' => '2.33333',
+					'43' => '1.33333',
+					'32' => '1.5',
+					'11' => '1',
+					'916' => '0.5625',
```
Comment (StyleShit) [inline includes/widgets/video.php:705]: maybe add a comment in each line?

```
// 16 / 9
```

**Review by StyleShit — CHANGES_REQUESTED** [review]

File: includes/widgets/video.php (hunk 2 above)
Comment (rami-elementor) [reply includes/widgets/video.php:705]: OK.

Comment (rami-elementor) [reply includes/widgets/video.php:705]:
```suggestion
					'169' => '1.77777', // 16 / 9
					'219' => '2.33333', // 21 / 9
					'43' => '1.33333', // 4 / 3
					'32' => '1.5', // 3 / 2
					'11' => '1', // 1 / 1
					'916' => '0.5625', // 9 / 16
```

**Review by StyleShit — DISMISSED** [review]

**Review by arielk — DISMISSED** [review]

**Comment by arielk** [comment]
@rami-elementor change to 3.13

**Review by arielk — APPROVED** [review]

**Pull request merged**